Elevel/Application/Features/ReportCommands/CreateReportCommand.cs

```diff
@@ -80,8 +84,46 @@ public Handler(IApplicationDbContext context, IMapper mapper)
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
+                Guid creatorId = Guid.Empty;
```

Comment:
you can directly set this field in report instead of having additional variable just for that

Reply:
Done